src/component/plasmic/shared/PlasmicActionBar.module.css

```diff
@@ -7,8 +7,9 @@
   flex-direction: column;
   padding-left: 0px;
   transition-property: all;
-  transition-duration: 1s;
+  transition-duration: 2s;
```

Comment:
Duration is based on the max-height so to keep the same speed when doubling (approximately) the max height (431->1000) we must double the duration.